tests/e2e/standard/test_cli.py

```diff
@@ -190,11 +187,11 @@ def test_clear_command_offline_and_online_containers(self, environment, containe
                 f"- {environment.project}/{container_sys_id}",
                 "",
                 "Unsynchronized offline objects:",
-                f"- offline/run__{offline_container_id}",
+                f"- offline/{offline_container_id}",
                 "",
                 "Do you want to delete the listed metadata? [y/N]: y",
-                f"Deleted: {offline_container_path}",
-                f"Deleted: {container_path_parent}",
+                f"Deleted: {offline_container_path.resolve()}",
```

Comment:
I decided not to have a prefix in `neptune` command-line tool. First of all, we will have to reformat a few of messages that are printing some lists etc + it doesn't look to bring any value in case of cli.